charts/rekor-tiles/rekor-tiles/templates/ingress.yaml

```diff
@@ -0,0 +1,81 @@
+{{- if .Values.ingress.enabled }}
+apiVersion: "networking.k8s.io/v1"
+kind: Ingress
+metadata:
+  labels:
+    {{- include "rekor-tiles.labels" . | nindent 4 }}
+  name: {{ template "rekor-tiles.fullname" . }}
+  annotations:
+{{ toYaml .Values.ingress.annotations | indent 4 }}
+spec:
+{{- if .Values.ingress.className }}
+  ingressClassName: {{ .Values.ingress.className }}
+{{- end }}
+  rules:
+    {{- range .Values.ingress.hosts }}
+    - host: {{ required "An Ingress hostname is required" .host | quote }}
+      http:
+        paths:
+          - path: {{ .path }}
+            pathType: {{ default "Prefix" .pathType }}
+            backend: {{- include "rekor-tiles.ingress.backend" $  | nindent 14 }}
+    {{- end }}
+{{- if .Values.ingress.tls }}
+  tls:
+    {{- range .Values.ingress.tls }}
+    - hosts:
+      {{- range .hosts }}
+      - {{ . | quote }}
+      {{- end }}
+      secretName: {{ .secretName }}
+    {{- end }}
+{{- end }}
+{{- end -}}
+{{- range .Values.ingresses }}
```

Comment:
We should collapse this template into a single one for `ingress`; the only reason there are both `ingress` and `ingresses` are when we made a transition from NGINX to GCE load balancers in pgi

Reply:
Done